app/views/locations/_form.html.erb

```diff
@@ -62,6 +62,9 @@
     if (blurredLocation.isBlurred()) tags = tags + ',' + 'location:blurred'
     if ($('#placenameDisplay').val() != 'Location unavailable') tags = tags + ',' + 'place:' + parameterize($('#placenameDisplay').val());
     addTag(tags);
+
+    <% reload="false" if reload.blank? %>
+    <% if reload == "true" %>setTimeout(function() { location.reload(); }, 3000);<% end %>
```

Comment:
Could we just say `<% if !reload.blank? && reload %>` here to compact it? Or, even better, was there a problem with just saying `<% if params[:reload] %>` and letting it read directly from the GET parameters in the URL?